paddle/majel/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cc_library(majel SRCS place.cc)
+cc_library(majel SRCS place.cc ddim.cc)
```

Comment:
Maybe we need to add an optional `OBJECT`, because CMake  `add_library(name OBJECT source)` can generate `*.o`